### ISSUE TITLE
Missing information on how to seed the Firestore Database?

### ISSUE THREAD

**Issue by oscarklm** [fireship-io/fkit-course#2]
Hey,

Is it on purpose, that there is no info in the course - nor any to be found on the repo, about setting up the database?

I'm thinking in terms of it already having data in the video. I think this could confuse quite a few people.

**Comment by nichemediapubs** [comment]
Agree. Did you make any progress with it @oscarklm ? I am stuck on it currently

**Comment by morgalorgadingdong** [comment]
I think you guys encountered the same hiccup I did. You do not have to manually add any data to your data base, but you do have to adjust the rules with the FireBase UI to allow read/write permissions. The most simple, and straightforward change to make to your rules would be to set "allow read, write: if false" to true. 

rules_version = '2';

service cloud.firestore {
  match /databases/{database}/documents {
    match /{document=**} {
      allow read, write: if true;
    }
  }
}

**Comment by jb-oe** [comment]
I'm also having issues with this. I have tried adding the rule above but it hasn't made any difference.

**Comment by vhazbun** [comment]
Same here, I don't se the users collection and usernames collection.

**Comment by dr-oofenshmirtz** [comment]
To add. [this](https://github.com/fireship-io/fkit-course/issues/2#issuecomment-1631332084) fixed my problem. It is done by navigating to "firestore database" in the left drawer, then selecting the rules tab and changing 'false' to 'true'

**Comment by krzysztofpodmokly** [comment]
This worked for me
```
rules_version = '2';

service cloud.firestore {
  match /databases/{database}/documents {
    match /{document=**} {
      allow read, write: if request.auth != null;
    }
  }
}
```

**Comment by jamesRoystonEvans** [comment]
Firebase auto selected my database the new datastore type. I had to switch to naitve before I could get any of the above to work. Then updating the rules as mentioned finally fixed the issues and the username check/creation is working.
